### PR TITLE
Edit comments

### PR DESCRIPTION
**Pull request by tuananhhoangit**
------------------------------------------
[x] Build Fail!
------------------------------------------

[o] rubocop: success
[x] rails_best_practices: failed
[o] eslint: success
[x] reek: failed
[o] rspec: success
[o] bundle-audit: success
[o] brakeman: success
[o] scss-lint: success

### CODE REVIEW

**Comment by tuananhhoangit** [comment]
ready

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+    end
   end
 
-  def destroy
-    @comment = @post.comments.find_by id: params[:id]
+  def update
+    if @comment.update_attributes comment_params
+      render json: {status: :success}
+    else
+      render json: {status: :error, html: "error"}
```
Comment (tuyenvd1177) [inline app/controllers/comments_controller.rb]: nội dung html là hiện thì ra thì i18n nhé

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -27,4 +36,13 @@ def load_post
 
     valid_info @post
   end
+
+  def load_comment
+    @post = Post.find_by id: params[:post_id]
+    @comment = @post.comments.find_by id: params[:id]
```
Comment (tuyenvd1177) [inline app/controllers/comments_controller.rb:42]: nil thì sao

**Comment by tuyenvd1177** [comment]
commented

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (tuananhhoangit) [reply app/controllers/comments_controller.rb:42]: đã sửa

**Comment by tuananhhoangit** [comment]
ready

**Pull request merged**